scripts/generate_dump

```diff
@@ -6,6 +6,15 @@
 
 set -u
 
+trap 'handle_signal' 2
+
+handle_signal()
+{
+    echo "Generate Dump received interrupt"
+    $RM $V -rf $TARDIR
```

Comment:
>$RM $V [](start = 4, length = 6)

These variables should be defined before this function.

Reply:
done